### PR TITLE
Fixed inconsistent spelling

### PR DESCRIPTION
**Pull request by agoryelov**

### CODE REVIEW

**Review by agoryelov — COMMENTED** [review]
LGTM

File: PerudoBot/Modules/LiarCommands.cs [hunk 1]
```diff
@@ -115,7 +115,7 @@ public async Task LiarAsync()
                 var penalty = (numberOfDiceMatchingBid - previousBid.Quantity) + 1; // if variable penalty
                 if (game.Penalty != 0) penalty = game.Penalty; // penalty is set to 0 for variable penalty games
 
-                if (PlayerEligebleForSafeguard(game.Penalty == 0, playerWhoShouldGoNext.NumberOfDice, penalty)) {
+                if (PlayerEligeebleForSafeguard(game.Penalty == 0, playerWhoShouldGoNext.NumberOfDice, penalty)) {
```
Comment (jonvandermey) [inline PerudoBot/Modules/LiarCommands.cs:118]: How dare you

**Review by jonvandermey — CHANGES_REQUESTED** [review]

File: PerudoBot/Modules/LiarCommands.cs (hunk 1 above)
Comment (emad-rahman) [reply PerudoBot/Modules/LiarCommands.cs:118]: What is the world is this abomination

**Comment by jonvandermey** [comment]
Was fixed in PR #8

**Pull request closed without merging**